src/library.rs

```diff
@@ -779,6 +783,15 @@ impl Type {
         }
     }
 
+    /// If the type is an Alias containing a fundamental, it'll return true (whereas
+    /// `is_fundamental` won't).
+    pub fn is_fundamental_type(&self, env: &Env) -> bool {
```

Comment:
Seems old `is_fundamental` used only internally so we can rename it to `is_fundamental_internal" (maybe make not public)

Reply:
But IMHO it good as is.